semcore/d3-chart/src/types/Line.d.ts

```diff
@@ -59,9 +59,33 @@ export type LineNullProps = Context & {
   hide?: boolean;
 };
 
+export type LineAreaProps = Omit<LineProps, 'transparent'> &
+  (
+    | {
+        /**
+         * Width of Area. Could be in percents (if < 1) or in values (if >= 1)
+         */
+        wide: number;
```

Comment:
What if we type it as `${number}${'px' | '%'}`? I think that total flip of behavior based on the value range is not the best practice we can follow